src/components/map/popups/MapPopupPilot.vue

```diff
@@ -90,6 +98,12 @@
                 @viewRoute="viewRoute()"
             />
         </template>
+        <template #depRunways>
```

Comment:
For the airport popup, the runway selector is in the ATC tab. For pilots, however, it is in the info tab. Perhaps, for consistency, it could also be moved to the ATC tab, but I am not sure about that. Having it in the main tab might be more convenient.

Reply:
Yeah we moved to ATC in airport because it would make no sense at all, but if we move it here also, people may not be able to find it... I guess I'll duplicate it!